0008-security-recommendations/README.md

```diff
@@ -0,0 +1,165 @@
+---
+name: KP Security Recommendations
+shortname: 8/KPSR
+status: raw
+editor: Alexey Shmalko <ashmalko@cybervisiontech.com>
+---
+## Introduction
+This document describes security recommendations for systems using 1/KP. This is only a recommendation and is not mandatory to implement.
+
+## Model
+We start with the generalization of the _physical_ system to better understand the attack surface, threats, and possible protection techniques.
+
+The general system consists of a server, clients, and end devices. End devices are devices that produce data, and the client only serves as a proxy for them. End devices might be represented as endpoints within 1/KP.
+
+Note that end device is always a real device, so end device is not always an endpoint. One end device might contain multiple endpoints.
+
+So the connection scheme looks like this:
+```
+server -- clients -- end devices
```

Comment:
End device is described inline. It could contain one or multiple endpoints but doesn't have to.